lua/starfall/libs_sh/find.lua

```diff
@@ -265,4 +265,12 @@ function find_library.playerBySteamID64(steamid)
 	if found then return plywrap(found) end
 end
 
+--- Returns entity that has given Entity:mapCreationID.
+-- @param number num Entity's creation id
+-- @return Entity Found entity
```

Comment:
Does this return NULL or nil if the entity isn't found? I think nil would be better, like the functions above it, and then this doc should be `-- @return Entity? The found entity or nil if not found`